doc/api/crypto.md

```diff
@@ -944,6 +944,15 @@ The `output_format` can specify one of `'latin1'`, `'hex'` or `'base64'`. If
 `output_format` is provided a string is returned; otherwise a [`Buffer`][] is
 returned.
 
+The optional `options` argument is an object which specifies additional
+cryptographic parameters. Currently, the following options are supported:
+
+* `padding` : {String} - RSA padding, either `'pkcs1'` for RSASSA-PKCS1-v1_5
+  (default) or `'pss'` for RSASSA-PSS
+* `saltLength` : {number} - salt length for RSASSA-PSS. If this is set to `-1`,
+  the salt length will be set to the digest size. If this is set to `-2`
+  (default), the salt length will be set to the maximum permissible value.
```

Comment:
> Should I just `#define RSA_PKCS1_PSS_MAX_SALT_LENGTH -2` and then use `NODE_DEFINE_CONSTANT`?

I don’t see any problem with that. :) It’s quite possible that somebody else here comes up with a better suggestion, but for the time being, I’d recommend just doing it the way you just described.